packages/vuetify/src/components/VBadge/VBadge.sass

```diff
@@ -5,6 +5,8 @@
   display: inline-block
   line-height: $badge-line-height
 
+  @include theme($badge-theme...)
```

Comment:
Shouldn't it just apply to the badge "dot"

Reply:
I agree. We should probably just integrate variants into badge later on.